packages/esm-patient-medications-app/src/components/medications-details-table.component.tsx

```diff
@@ -131,6 +139,9 @@ const MedicationsDetailsTable: React.FC<ActiveMedicationsProps> = ({
                 {formatDate(new Date(medication.dateStopped))}
               </span>
             ) : null}
+            {isMedicationDiscontinued(medication) && !isMedicationModified(medication) && (
```

Comment:
Just inline the function definitions. Don't write one-line functions.

Reply:
@brandones Oh sorry for the late response!  I have been  sick that is the reason of my late response. The reason it was in draft is because I was informed the design was still in discussion.

Reply:
@brandones am on it right now.